src/Matterhorn/State/Messages.hs

```diff
@@ -681,6 +686,57 @@ runNotifyCommand post mentioned = do
                                  [notified, sender, messageString] Nothing Nothing
                 return Nothing
 
+encodeToJSONstring :: A.ToJSON a => a -> String
+encodeToJSONstring a = map (chr . fromEnum) . BL.unpack $ A.encode a
+
+-- We define a notifyGetPayload for each notification version.
+notifyGetPayload :: Int -> ChatState -> Post -> Bool -> MH (Maybe String)
+
+-- Notification Version 2
+data NotificationV2 = NotificationV2
+    { version :: Int
+    , message :: Text
+    , mentioned :: String
```

Comment:
If I recall, this is a boolean value. If that's right, then I think this field would be better as a `Bool` with a conversion to the `yes`/`no` values moved to the en/decoder.

Reply:
In fact, really, it would be okay to just put a bool in the JSON data rather than encoding as a string.

Reply:
Resolved by commit 6553fb5b .

Reply:
Resolved by commit 6553fb5 .